account/models.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+
+from django.db import models
+
+from django.contrib.auth import get_user_model
+
+
+class Account(models.Model):
+    type_of_currency = models.CharField(max_length=20)
+    user = models.ForeignKey(get_user_model(), related_name='user_account',
```

Comment:
This line creates relation OneToMany here. It means that one user can to have many accounts. 
Is it ok here? Or you just want to use OneToOne relationship when one user has only one account?